initfiles/componentfiles/thor/stop_thor

```diff
@@ -47,22 +47,9 @@ if [ "$#" -lt "2" ] || [ "$2" != "keep_sentinel" ]; then
     sleep 1
 fi
 
-masterpid=`cat ${MASTER_PID_NAME} 2> /dev/null`
-if [ ! -z $masterpid ]; then
-  while :
-  do
-    kill -0 $masterpid >& /dev/null
-    masterRunning=$(( $? == 0 ? 1 : 0 ))
-    if [ 0 == $masterRunning ]; then
-      break
-    fi
-    echo --------------------------
-    echo stopping thormaster $THORMASTER 
-    kill $masterpid >& /dev/null
-    sleep 8
-    kill -9 $masterpid >& /dev/null
-    sleep 1
-  done
+kill_process ${MASTER_PID_NAME} 8
```

Comment:
That's not what I was expecting.. 
previously it was just checking the pid (and in theory an unrelated process could take it whilst it was looping)
Now it only takes the process name.. I was expecting a utility method that took name and pid.. and verified the pid it was told to kill was indeed the name of the process it was told.
It may suffice, if MASTER_PID_NAME is unique.. but in general I think it would be better if checked.